style.css

```diff
@@ -619,25 +579,197 @@ body {
   font-weight: 700;
   font-size: 40px;
   line-height: 52px;
-  margin-left: 50px;
-  margin-bottom: 20px;
 }
 
 .contact-text {
   font-style: normal;
-  font-weight: 400;
   font-size: 20px;
   line-height: 28px;
-  text-align: center;
   color: #ebebff;
-  margin-bottom: 40px;
-}
-
-.contact-form-background-image {
-  position: absolute;
-  width: 233px;
-  height: 670.17px;
-  left: 153px;
-  margin-left: -10px;
-  margin-top: 49px;
+  margin-right: 39px;
+  padding-right: 30px;
+  font-weight: 400;
+  width: 569px;
+  height: 102px;
+}
+
+/* ------------Desktop Version----------- */
+
+@media screen and (min-width: 768px) {
+  .body {
+    width: 90%;
+    background: #fff;
+  }
+
+  .toolbar {
+    display: flex;
+  }
+
+  .icons {
+    display: none;
+  }
+
+  .navu {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 90%;
+  }
+
+  .navul1 {
+    margin-left: 150px;
+    margin-top: -20px;
+  }
+
+  .navul2 {
+    margin-top: -20px;
+    margin-left: 700px;
+    width: 50%;
+  }
+
+  .navulitems {
+    font-style: normal;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+    display: flex;
+    flex-direction: row;
+    list-style-type: none;
+    padding-right: 100px;
+  }
+
+  .navli {
+    color: #344563;
+    margin: 5px;
+  }
+
+  .tool {
+    background-image: url(./Header\ bg.png);
+  }
+
+  .headline {
+    display: flex;
+    flex-direction: column;
+    width: 90%;
+    align-items: flex-start;
+    background-image: no-repeat;
+  }
```

Comment:
Great work on the website so far. However there are few things you need to work on. 

- [ ] There is a border radius at the top left corner of the original desing which is missing from yours. Kindly check and update it accordingly. :wink: 
- [ ] Kindly check the position and coverage area of the background picture as well as the font-size and line-height of the text in the headline section. Ensure that they match the original design since there are some small variations. :pray: 

Your design 

![Screenshot from 2022-12-15 21-54-49](https://user-images.githubusercontent.com/65233860/207976199-7a1d8414-9181-4a23-a95d-bdde93d18d7c.png)


The original design


![Screenshot from 2022-12-15 21-42-06](https://user-images.githubusercontent.com/65233860/207976233-9db0f99f-8643-467a-9d79-e140932d3b65.png)